rcode-reader/src/main/java/eu/rssw/pct/elements/QueryElement.java

```diff
@@ -8,29 +8,37 @@
 import eu.rssw.pct.RCodeInfo;
 
 public class QueryElement extends AbstractAccessibleElement {
-  protected int prvte;
-  protected int cacheSize;
-  protected int flags;
-  protected int crc;
-  private String[] bufferNames;
+  private final String[] bufferNames;
+  private final int prvte;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "prvte" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1068)